package.nls.json

```diff
@@ -9,6 +9,7 @@
     "containerApps.editContainerApp": "Edit Container App (Advanced)...",
     "containerApps.deployImage": "Update Container App Image...",
     "containerApps.deployImageApi": "Update Container App Image (API)...",
+    "containerApps.deployWorkspaceProject": "Deploy Project from Workspace...",
```

Comment:
Should we call this `Create Container App from Workspace...`?

My thought process is that the user may infer that "deploy" means that they need to have an ca already created (especially since it's next to a create command)

Reply:
And following that logic, maybe we should rename `Create Container App...` to something like `Create Container App from Remote/Registry...` or

Reply:
Maybe worth discussing with the CA team to see if there is a better name to use here.  I still kind of prefer `deploy` because it's more accurate than create, because this can also re-deploy to an existing

Reply:
Yeah, it's a small nuance but I think we should do something about `Create Container App...` then. I have a bad feeling people may try to do that _first_ and see how many steps it is and then just give up.